### ISSUE TITLE
Indexing always fails with Xcode 10.2

### ISSUE THREAD

**Issue by salutis** [peripheryapp/periphery#28]
From #8:

> +1 with @Sam-Spencer 
> 
> I've just compiled the `xcode10.2` branch, and Periphery failed with the following error:
> 
> ```
> * Inspecting project configuration...
> * Building Acrostics...
> * Indexing...
> objc[30286]: __SwiftNativeNSError object 0x7f85eb111ab0 overreleased while already deallocating; break on objc_overrelease_during_dealloc_error to debug
> error: SourceKit editorOpen request failed for file '[REDACTED]': Retrieving both the document structure and a syntax tree at the same time is not supported. Use the syntax tree to compute the document structure.
> ```
> 
> This is still broken.

**Comment by nysander** [comment]
I have also errors while indexing on Xcode 10.2:

```
periphery scan \
  --workspace "......workspace.xcworkspace/" \
  --schemes "Scheme" \
  --targets "Target","TargetTests" \
  --retain-public

* Inspecting project configuration...
* Building Target...
* Indexing...
error: SourceKit editorOpen request failed for file '...../R.generated.swift': 
Retrieving both the document structure and a syntax tree at the same time 
is not supported. Use the syntax tree to compute the document structure.
```
the same appears on other tries for other files in my project

**Comment by Trifusion** [comment]
Same issue with 10.2:

periphery scan \
  --project "XXXXX" \
  --schemes "XXXX" \
  --targets "XXX" \
  --retain-public

* Inspecting project configuration...
* Building XXXX...
* Indexing...
error: SourceKit editorOpen request failed for file 'XXXXX/XXX/XXX.swift': Retrieving both the document structure and a syntax tree at the same time is not supported. Use the syntax tree to compute the document structure.

**Comment by paulhargreaves** [comment]
Also same:

```
* Inspecting project configuration...
* Building <mine>...
* Indexing...
error: SourceKit editorOpen request failed for file '<mine>.swift': Retrieving both the document structure and a syntax tree at the same time is not supported. Use the syntax tree to compute the document structure.
```

**Comment by Trifusion** [comment]
Same issue with latest 10.2.1 XCode update

**Comment by chiliec** [comment]
I have the same issue, XCode version 10.2.1 (10E1001)

**Comment by tantakan** [comment]
Looks not released for Xcode 10.2.x (actually periphery 1.4.0 build in Xcode 10.1).
So now works with 10.2.1 (10E1001) in local build.

Do you have any plans for periphery 1.5.0 with Xcode 10.2.x builds?

**Comment by ZhangComer** [comment]
Still facing the same issue with Xcode (10.2.1) and periphery (1.4.0).

**Comment by sirnacnud** [comment]
I'm running in to this issue as well, periphery 1.4.0 and Xcode 10.2.1

**Comment by iwheelbuy** [comment]
Same here. Tried from latest master

**Comment by ileitch** [comment]
Fixed in 1.5.1.

**Comment by nysander** [comment]
confirm. works now :) thank you
